src/magic.cpp

```diff
@@ -315,6 +317,8 @@ void spell_type::load( const JsonObject &jo, const std::string & )
     spell_class = trait_id( temp_string );
     optional( jo, was_loaded, "energy_source", temp_string, "NONE" );
     energy_source = energy_source_from_string( temp_string );
+    optional( jo, was_loaded, "energy_item", temp_string, "NONE" );
+    energy_req_item = temp_string;
```

Comment:
You can read energy_req_item directly, no need to use `temp_string` here. The other code is doing it that way because the values need to be converted to enums.